android/build.gradle

```diff
@@ -90,16 +90,16 @@ repositories {
 
       if (androidPrebuiltBinaryDir.exists()) {
         maven {
-          url androidPrebuiltBinaryDir.toString()
```

Comment:
this syntax is currently deprecated and will be removed in a future gradle release.